shifter/theme/templates/base.html

```diff
@@ -2,79 +2,120 @@
+                            <ul class="py-1 text-sm text-gray-700" aria-labelledby="dropdownLargeButton">
+                                <li>
+                                    <a href="{% url 'shifter_files:myfiles' %}"
+                                        class="block py-2 px-4 hover:bg-gray-100">My Files</a>
+                                </li>
+                                <li>
+                                    <a href="{% url 'shifter_auth:settings' %}"
+                                        class="block py-2 px-4 hover:bg-gray-100">Settings</a>
+                                </li>
+                                {% if user.is_staff %}
+                                <li>
+                                    <a href="{% url 'shifter_auth:create-new-user' %}"
+                                        class="block py-2 px-4 hover:bg-gray-100">Register New User</a>
+                                </li>
+                                {% endif %}
+                            </ul>
+                            <div class="py-1">
                                 <form method="post" action="{% url 'shifter_auth:logout' %}">
                                     {% csrf_token %}
-                                    <input type="submit" value="Sign Out" class="block py-2 px-4 text-sm text-gray-700 hover:bg-gray-100 cursor-pointer w-full text-left">
+                                    <input type="submit" value="Sign Out"
+                                        class="block py-2 px-4 text-sm text-gray-700 hover:bg-gray-100 cursor-pointer w-full text-left">
                                 </form>
-                                </div>
                             </div>
-                        </li>
-                        {% else %}
-                        <li>
-                            <a href="{% url 'shifter_auth:login' %}" class="flex">
-                                <span class="btn-primary">Sign In</span>
-                            </a>
-                        </li>
-                        {% endif %}
-                    </ul>
-                </div>
+                        </div>
+                    </li>
+                    {% else %}
+                    <li>
+                        <a href="{% url 'shifter_auth:login' %}" class="flex">
+                            <span class="btn-primary">Sign In</span>
+                        </a>
+                    </li>
+                    {% endif %}
+                </ul>
             </div>
-        </nav>
-        <div class="container mx-auto mt-10">
-            <div class="flex justify-center">
-                {% block content %}
-                <div class="flex items-center justify-center">
-                    <h1 class="text-5xl">Template Page</h1>
-                </div>
-                {% endblock %}
+        </div>
+    </nav>
+    <div class="container mx-auto mt-10">
```

Comment:
Can you add the `flex-grow` class to this div:
```
<div class="container mx-auto mt-10 flex-grow">
```
to make the footer fix to the bottom of the screen.